File: public/nootan-20160520092127/JAVA/sherlock.java
// Name : Sherlock
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : size
		// m : size
		// map : ratio
		int idx, jdx;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int m = Integer.parseInt(elements[1]);
		char[][] map = new char[][];
		String map_elements = "";
		for(idx=0;idx<n;idx++){
			map_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<m;jdx++){
				map[idx][jdx] = map_elements[jdx];
			}
		}

		// write your code here
		// store your results in `row`,`column`

		// output
				System.out.println(row+" "+column);

	}
}

File: public/nootan-20160520092127/JAVASCRIPT/sherlock.js
// Name : Sherlock
// input:
// n : size
// m : size
// map : ratio
var idx,jdx;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var m = parseInt(elements[1], 10);
var map = new Array([]);
var map_elements = "";
for(idx=0;idx<n;idx++){
	map_elements = readline().split(' ');
	map[idx] = new Array([]);
	for(jdx=0;jdx<m;jdx++){
		map[idx][jdx] = map_elements[jdx];
	}
}

// write your code here
// store your results in `row`,`column`

// output
print(row+" "+column);
